docs/testnet/onomy-testnet-docs/setting-up-a-fullnode-manual.md

```diff
@@ -0,0 +1,79 @@
+# How to run a Onomy testnet full node
+
+A Onomy chain full node is just like any other Cosmos chain full node and unlike the validator flow requires no external software
+
+## What do I need?
+
+A Linux server with any modern Linux distribution, 2gb of ram and at least 20gb storage. Requirements are very minimal.
+
+### Download/install Onomy chain binaries
+```
+To download binary follow these commands
+mkdir binaries
+cd binaries
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/onomyd
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/gbt
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/geth
+cd ..
+chmod -R +x binaries
+export PATH=$PATH:$HOME/binaries/
+
+
+or If you have Fedora (Fedora 34) or Redhat (Red Hat Enterprise Linux 8.4 (Ootpa))
```

Comment:
Agreed, I was about to make the same statement. I'd also add, for the first block:
`cd ~`

To ensure you're in home, otherwise the final export may not work.

Reply:
All these suggestion has been implemented